Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -319,6 +320,42 @@ theorem apply_one (x : R) : (1 : MulRingNorm R) x = if x = 0 then 0 else 1 :=
 instance : Inhabited (MulRingNorm R) :=
   ⟨1⟩
 
+
+variable {R : Type*} [Ring R]
+
+/-- Two multiplicative ring norms `f, g` on `R` are equivalent if there exists a positive constant
+  `c` such that for all `x ∈ R`, `(f x)^c = g x`.
+  This could be generalised to RingNorm, but MulRingNorm does not extend this. -/
```

Comment:
I am not sure I understand the sentence "This could be generalised to RingNorm, but MulRingNorm does not extend this". Do you plan to do this?

Reply:
Yes, this is an old comment from the Lean 3 version of the project. We don't need to generalize it for Ostrowski, though, so we implemented the case we needed.